web-client/iron-remote-desktop-rdp/pre-build.js

```diff
@@ -6,7 +6,7 @@ let run = async function (command, cwd) {
             stdio: 'pipe',
             shell: true,
             cwd: cwd,
-            env: { ...process.env, RUSTFLAGS: '-Ctarget-feature=+simd128' },
+            env: { ...process.env, RUSTFLAGS: '-Ctarget-feature=+simd128,+bulk-memory' },
```

Comment:
note: I think we have a similar env variable in xtask::web. Can you also update at this place?

Reply:
Done